Workload-Security/Integration/aws-python-amazon-inspector-integration/c1-inspector-findings-protection.yaml

```diff
@@ -0,0 +1,837 @@
+AWSTemplateFormatVersion: 2010-09-09
+Parameters:
+  AwsRegion:
+    Type: String
+    Description: Please enter the region in which the lambda will be executed
+    Default : us-east-1
+    AllowedValues:
+      - us-east-1
+      - us-east-2	
+      - us-east-1	
+      - us-west-1	
+      - us-west-2
+      - af-south-1
+      - ap-east-1
+      - ap-southeast-3
+      - ap-south-1
+      - ap-northeast-3
+      - ap-northeast-2
+      - ap-southeast-1
+      - ap-southeast-2
+      - ap-northeast-1
+      - ca-central-1
+      - eu-central-1
+      - eu-west-1
+      - eu-west-2
+      - eu-south-1
+      - eu-west-3
+      - eu-north-1
+      - me-south-1
+      - me-central-1
+      - sa-east-1
+
+  ApiKey:
+    Type: String
+    NoEcho: "true"
+    Description: Please enter your Cloud One API Key
+  EmailSender:
+    Type: String
+    Description: Email address to be used for sending report
+  EmailRecipients:
+    Type: String
+    Description: Comma separated (without spaces) email addresses for recieving report
+  CronSchedule:
+    Type: String
+    Description: Schedule the time you would like the lambda function to run in the cron format (Default is every wednesday)
+    Default: cron(0 12 ? * WED *)
+  CloudOneRegion:
+    Type: String
+    Description: Enter your Cloud One Region
+    Default: us-1
+    AllowedValues:
+      - trend-us-1
+      - us-1
+      - in-1
+      - gb-1
+      - jp-1
+      - de-1
+      - au-1
+      - ca-1
+      - sg-1
+Resources:
+  LambdaLayer1:
+    Type: "AWS::Lambda::LayerVersion"
+    Properties:
+      CompatibleRuntimes:
+        - python3.9
+      Content:
+        S3Bucket: vulnerabilitytestbucket
+        S3Key: "python.zip"
```

Comment:
Unless you can control under this cfn how the zip is being generated, I don't think is a good idea to hardcode it...